contributing/contributing_to_datacommons.md

```diff
@@ -21,18 +29,42 @@ You generally only need to submit a CLA once, so if you've already submitted one
 (even if it was for a different project), you probably don't need to do it
 again.
 
-## Code reviews
+### Becoming familiar with the project
+
+You can find suggested reading about the domain in which Data Commons operates in [the Background section](/contributing/background/background.html).
 
-All submissions, including submissions by project members, require review. We
-use GitHub pull requests for this purpose. Consult
+All submissions, including submissions by project members, require review. Data Commons
+uses GitHub pull requests for this purpose. Consult
 [GitHub Help](https://help.github.com/articles/about-pull-requests/) for more
 information on using pull requests.
 
-## Style guide
+This project follows Google's [code style guide](https://google.github.io/styleguide/), [documentation style guide](https://developers.google.com/style), and [open source community guidelines](https://opensource.google/conduct/).
+
+## Your first contribution
+
+### Contributing a dataset
+
+Directions are available on [the 'Adding Datasets' page](/contributing/adding_datasets.html).
+
+### Creating a new tool
+
+Data Commons welcomes the development of new tools that make the data on Data Commons more accessible to end users. If you have developed or would like to propose a new tool, feel free to create a PR in [the Data Commons tools repo](https://github.com/datacommonsorg/tools) or contact the team at <support@datacommons.org>.
+
+### Sharing analysis
+
+Data Commons is currently posting example analyses on [the tutorials page](/tutorials.html). To submit your analysis as a tutorial, open a PR in [the Data Commons documentation website repo](https://github.com/datacommonsorg/docsite/pulls) with the title of the analysis and links to its Colab notebook and Github raw .ipynb file. Your addition to the tutorial page should look like this when raw:
+
+```
+-   ** *YOUR TITLE HERE* **
```

Comment:
Maybe use a different placeholder text indicator, since ** is for bold?

Reply:
So I was trying to follow https://developers.google.com/style/placeholders, which suggests that placeholders should be marked with asterisks. However, I agree with you that this is odd as a placeholder indicator. Would it make more sense to mark the placeholder as `<YOUR TITLE HERE>`?

Reply:
SG

Reply:
fixed!

Reply:
thanks!